patches/rules_apple_module_name.patch

```diff
@@ -0,0 +1,84 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Derek Ostrander <derko@squareup.com>
+Date: Fri, 9 Dec 2022 13:55:16 -0500
+Subject: Add product module name as an optional arg for the test rule
```

Comment:
What's the longer term solution for this one? It'll likely be painful to maintain this longer term. Someone should either:
1. propose upstream fixes to it
2. copy the rules_apple code we need for these rules into rules_ios